content/broker/broker-transition/getting-started/_index.en.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Getting started with Altinn Broker Transition Solution
+linktitle: Getting Started
+description: How to get started using the Broker Transition solution from Altinn 2 to Altinn 3
+tags: [solution, broker, transition, guide]
+toc: false
+weight: 1
+---
+
+## ServiceOwners
+In order to use the Broker Transition solution in Altinn 2 to create, upload and retrieve file metadata, a service owner must complete the following steps.
+1. Have an existing Altinn 2 Broker Service.
+2. Have or create a corresponding Altinn 3 Broker Resource. 
+See how to get started with Altinn Broker [here](../../getting-started/).
+1. <!-- TODO: Set up of rights for the Altinn 3 Resource>
```

Comment:
Consider if these TODO-statements could be removed.
Also; the first ome on line 15 has a numbered list formatting, leading to an empty "point 3" in this list.